js/index.js

```diff
@@ -1,24 +1,213 @@
+    online: false,
+  },
+  work4: {
+    online: false,
+  },
+};
+const worksDesktop = {
+  work1: {
+    name: 'Tonic',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: '../images/Desktop-Snapshoot Portfolio1.jpg',
+    technologies: ['html', 'css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    linkLiveVersion: '',
+    linkSource: '',
+  },
+  work2: {
+    online: true,
+  },
+  work3: {
+    online: false,
+  },
+  work4: {
+    online: false,
+  },
+};
+
+function technologies() {
+  const technologiesList = createelement('ul', 'card-works-technologies');
+
+  if (window.screen.width < 768) {
+    for (let i = 0; i < worksMobile.work1.technologies.length; i += 1) {
+      const technologiesItem = createelement('li', 'card-works-technologies-item');
+      const liText = document.createTextNode(worksMobile.work1.technologies[i]);
+      technologiesItem.appendChild(liText);
+      technologiesList.appendChild(technologiesItem);
+    }
+  } else {
+    for (let i = 0; i < worksDesktop.work1.technologies.length; i += 1) {
+      const technologiesItem = createelement('li', 'card-works-technologies-item');
+      const liText = document.createTextNode(worksDesktop.work1.technologies[i]);
+      technologiesItem.appendChild(liText);
+      technologiesList.appendChild(technologiesItem);
+    }
+  }
+  return technologiesList;
+}
+
+function createWorkDetails() {
+  shutdownBackground2();
+  const container = document.querySelector('#container');
+  const menu = createelement('div', 'work-details');
+  container.appendChild(menu);
+  const cardWorks = createelement('div', 'card-works');
+  menu.appendChild(cardWorks);
+  cardWorks.appendChild(createelement('div', 'card-works-snapshot'));
+  const buttonClose = createelement('button', 'card-works-close');
+  buttonClose.addEventListener('click', restoreView2);
+  cardWorks.appendChild(buttonClose);
+  cardWorks.appendChild(createelement('h2', 'card-works-project-title'));
+  cardWorks.appendChild(createframe2());
+  cardWorks.appendChild(createelement('p', 'card-works-description'));
```

Comment:
- [ ] Great job createing the Pop up but I have a samll note on the mobile version I can see the skills are aoutsiide the pop up so please make sure to style them well to be not outside the pop up you can use flex-wrap to make them under each other but to not be outside 💪🏻🔥📌💯


## On the Desktop version
- [ ] Great job working on the pop up for the desktop version but I think there is an empty space or maybe an mepty div to please try to get rig ot it because it makes your Portfolio not like the Figma design 💯📌👍🏻

![image](https://user-images.githubusercontent.com/58553711/197086234-56e22825-5605-4ec4-b062-ff07ca279cb0.png)